src/Integration/Telemetry/TelemetryManager.cs

```diff
@@ -46,7 +47,26 @@ public TelemetryManager(ISlCoreTelemetryHelper telemetryHelper, IKnownUIContexts
         knownUiContexts.VBProjectContextChanged += OnVBProjectContextChanged;
     }
 
-    public void QuickFixApplied(string ruleId) => telemetryHelper.Notify(telemetryService => telemetryService.AddQuickFixAppliedForRule(new AddQuickFixAppliedForRuleParams(ruleId)));
+    public void QuickFixApplied(string ruleId) =>
+        telemetryHelper.Notify(telemetryService =>
+            telemetryService.AddQuickFixAppliedForRule(new AddQuickFixAppliedForRuleParams(ruleId)));
+
+    public void FixSuggestionApplied(string suggestionId, IEnumerable<bool> changeApplicationStatus) =>
```

Comment:
This takes care of `Applied` and `Declined` changes, the method can be renamed to `Resolved`. It will also be consistent with the SLCore naming.

```suggestion
    public void FixSuggestionResolved(string suggestionId, IEnumerable<bool> changeApplicationStatus) =>
```